docs/getting-started.md

```diff
@@ -16,6 +16,30 @@ http.createServer((req, res) => {
 
 ```
 
+You will need to also provide a minimal `package.json` with a `start` script. When `up` detects a `package.json`
```

Comment:
hmm this part isn't true, all you need is an `app.js`, and the "start" script is picked up only when present, otherwise it defaults to `node app.js`